src/three-components/Renderer.ts

```diff
@@ -237,13 +237,13 @@ export class Renderer extends EventDispatcher {
       context.drawImage(
           this.renderer.domElement,
           0,
-          this.canvas.height - heightDPR,
+          this.canvas3D.height - heightDPR,
           widthDPR,
           heightDPR,
           0,
           0,
-          widthDPR,
-          heightDPR);
+          width,
+          height);
```

Comment:
I think this will be fine for effective DPRs of `1`, but on my MBP we get this predictable result:

Canvas pixels w/ transparent space |
---|
![image](https://user-images.githubusercontent.com/240083/68148908-c2c87000-fef1-11e9-8a57-90f6556cf4e5.png)|

You have to take DPR into account when drawing to the 2D canvas context, as the base canvas width/height in pixels is scaled by DPR.

Reply:
Reverted.